ERCS/erc-xxxx.md

```diff
@@ -0,0 +1,201 @@
+---
+eip:
+title: Set of Wallet Capabilities for Account Abstraction Applications
+description: A way for apps and Account Abstraction wallets to communicate more advanced parameters of Account Abstraction operations
+author: Yoav Weiss (@yoavw), Alex Forshtat (@forshtat), Dror Tirosh (@drortirosh), Shahaf Nacson (@shahafn)
+discussions-to:
+status: Draft
+type: Standards Track
+category: ERC
+created:
+requires: 5792, 7702
+---
+
+## Abstract
+
+[EIP-5792](./eip-5792) defines a baseline JSON-RPC API for a communication between wallets and dapps.
+With EIP-5792, apps and wallets can communicate about any advanced features using "capabilities" - extensions
+to the base protocol that must be defined in separate documents.
+
+This proposal defines a set of "capabilities" the wallets may want to implement in order to provide a
+comprehensive support for Account Abstraction.
+
+## Motivation
+
+## Specification
+
+### Shared configuration
+
+All actions in Account Abstraction within the context of EIP-5792 must be done on a single chain and atomically.
+This means all requests to the `wallet_sendCalls` methods MUST be done:
+
+1. With the `atomicBatch` capability enabled
+2. With the `chainId` set to the same value in all calls
+
+### Static Paymaster Configuration Capability
+
+Note that use of Paymasters managed by a "paymaster web service" is described in [ERC-7677](./eip-7677).
+
+Identifier:
+
+`staticPaymasterConfiguration`
+
+Interface:
+
+```typescript
+type StaticPaymasterConfigurationCapabilityParams = Record<
+  `0x${string}`, // Chain ID
+  {
```

Comment:
this paymaster is version-specific: e.g. the app has to know whether the account is 4337v6, v7 or 7560, and based on that knowledge, select the right paymaster address

Reply:
That means we need something like a `aa_getAccountDetails` API, doesn't it?